src/rp2040/Kconfig

```diff
@@ -0,0 +1,109 @@
+# Kconfig settings for STM32 processors
+
+if MACH_RP2040
+
+config RP2040_SELECT
+    bool
+    default y
+    select HAVE_GPIO
+    select HAVE_CHIPID
+
+config BOARD_DIRECTORY
+    string
+    default "rp2040"
+
+config MCU
+    string
+    default "rp2040"
+
+config CLOCK_FREQ
+    int
+    default 12000000
+
+config FLASH_SIZE
+    hex
+    default 0x200000
+
+config RAM_START
+    hex
+    default 0x20000000
+
+config RAM_SIZE
+    hex
+    default 0x42000
+
+config STACK_SIZE
+    int
+    default 512
+
+config FLASH_START
+    hex
+    default 0x10000000
+
+config APPLICATION_START
+    default 0x10004000
```

Comment:
16kb is reserved for the bootloader. The current bootloader  size is 5kb for usb, 7 kb for can.